scripts/install_kubebuilder.sh

```diff
@@ -8,13 +8,18 @@ KUBEBUILDER_BIN=${KUBEBUILDER_BIN:=/usr/local/kubebuilder}
 OS=$(go env GOOS)
 ARCH=$(go env GOARCH)
 KUBEBUILDER_RELEASE=2.3.1
```

Comment:
this URL change is only valid for 3.0.0+ so this line would need to change. https://github.com/kubernetes-sigs/kubebuilder/releases/download/v3.3.0/kubebuilder_linux_amd64 works but for 2.3.1 it's https://github.com/kubernetes-sigs/kubebuilder/releases/download/v2.3.1/kubebuilder_2.3.1_linux_amd64.tar.gz (an extra version in the name + the tar.gz

Reply:
Yeah, that's basically the change I've made. I've put a note to say that from 3.0.0+ the url wouldn't work. This is to unblock CI. Once this is through I can look at upping the version to 3.0.0 as a different issue. wdyt?